base_layer/wallet/src/key_manager_service/handle.rs

```diff
@@ -44,6 +45,9 @@ pub struct KeyManagerHandle<TBackend> {
 impl<TBackend> KeyManagerHandle<TBackend>
 where TBackend: KeyManagerBackend + 'static
 {
+    /// Creates a new Key manager and wrap this inside of the handle
+    /// * `master_seed` is the primary seed that will be used to derive all unique branch keys with their indexs
+    /// * `db` is the key manager backed to be used for persistent storage of branches and indices.
```

Comment:
The fact that it wraps something else is an implementation detail so I would leave it out. 
A handle in Tari usually refers to something that sends messages to a service, so it should probably be renamed at some point in another PR. 
```suggestion
    /// Creates a new key manager.
    /// * `master_seed` is the primary seed that will be used to derive all unique branch keys with their indexes
    /// * `db` implements `KeyManagerBackend` and is used for persistent storage of branches and indices.
```